pkg/profilecreator/profilecreator.go

```diff
@@ -255,8 +256,37 @@ func (ghwHandler GHWHandler) SortedTopology() (*topology.Info, error) {
 	return topologyInfo, nil
 }
 
+func (ghwHandler GHWHandler) TopologyHTDisabled(info *topology.Info) *topology.Info {
```

Comment:
this should be a free function: we never use `ghwHandler`, do we?

Reply:
Right! Will update.